File: benchmarks/count-1M/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.nanoTime();

        int count = 0;
        for (int i = 0; i < 1_000_000; i++) {
            count++;
        }

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}

File: benchmarks/count-1M/main.go
package main

import "fmt"
import "time"

func main() {
	var before = time.Now().UnixNano()

	var count = 0
	for i := 0; i < 1000000; i++ {
		count++
	}

	var after = time.Now().UnixNano()

	fmt.Println(after - before)
}
